src/Volume.ts

```diff
@@ -244,19 +246,32 @@ export default class Volume {
     let noReload =
       this.loadSpec.time === this.loadSpecRequired.time &&
       this.loadSpec.subregion.containsBox(this.loadSpecRequired.subregion) &&
-      this.loadSpecRequired.channels.every((channel) => this.loadSpec.channels.includes(channel));
+      this.loadSpecRequired.channels.every((channel) => this.loadSpec.channels.includes(channel)) &&
+      this.loadSpecRequired.scaleLevelBias === this.loadSpec.scaleLevelBias;
```

Comment:
Kinda same comment as below: maybe it's the GitHub wrapping here, but I'm having trouble following what this is doing